examples/plot_0_first_demo.py

```diff
@@ -135,14 +135,15 @@ def generate_random_walk(NUM_CHANNELS, TIME_DATA_SAMPLES):
 # We are now ready to go to instantiate the *Stream* and call the *run* method for feature estimation:
 
 stream = nm.Stream(
+    data=data,
```

Comment:
I think putting the `data` here makes sense. Especially from the perspective of the GUI.
So in the first `setup` page we define those parts: `data` or `LSL stream name`, `output path`, `run name`. And then at the end of the settings page we really just have a `start stream / run` button.